### PR TITLE
VB-14: Create theme

### PR DESCRIPTION
**Pull request by dannleed**
Added color palette to shared and theme to frontend.

### CODE REVIEW

File: packages/frontend/components/layout/component.tsx [hunk 1]
```diff
@@ -10,6 +11,10 @@ export const Layout: FC<LayoutProps> = ({ title, children }) => (
   <>
     <Head>
       <title>{getTitle(title)}</title>
+      <style>
```
Comment (Vitalii4as) [inline packages/frontend/components/layout/component.tsx:14]: Isn't it easier to use <link>?

Comment (Nightlngale) [reply packages/frontend/components/layout/component.tsx:14]: Also move this link to the _document

Comment (dannleed) [reply packages/frontend/components/layout/component.tsx:14]: thanks, youre right, done

File: packages/frontend/pages/_app.tsx [hunk 2]
```diff
@@ -2,11 +2,12 @@ import { NextUIProvider } from '@nextui-org/react';
 import type { AppProps } from 'next/app';
 import { wrapper } from 'store';
 import { setGlobalStyles } from 'theme';
+import { lightTheme } from 'theme/theme';
```
Comment (Nightlngale) [inline packages/frontend/pages/_app.tsx:5]: Let's import lightTheme from theme

**Review by Nightlngale — APPROVED** [review]

File: packages/shared/src/common/enums/theme/color-palette.ts [hunk 3]
```diff
@@ -0,0 +1,14 @@
+enum ColorPalette {
```
Comment (Vitalii4as) [inline packages/shared/src/common/enums/theme/color-palette.ts:1]: Use UPPER_CASE for constants

Comment (dannleed) [reply packages/shared/src/common/enums/theme/color-palette.ts:1]: fixed

**Review by Nightlngale — APPROVED** [review]

**Pull request merged**